<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main Window</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>356</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCreate"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSet_Color_Window"/>
    <addaction name="actionSet_Color_Text"/>
   </widget>
   <widget class="QMenu" name="menu_Language">
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Language</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_About_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_Language"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionCreate"/>
   <addaction name="action_About"/>
   <addaction name="action_About_Qt"/>
   <addaction name="actionSet_Color_Window"/>
   <addaction name="actionSet_Color_Text"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCreate">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/icon/document.ico</normaloff>:/new/prefix1/icon/document.ico</iconset>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/icon/folder.ico</normaloff>:/new/prefix1/icon/folder.ico</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/icon/delete.ico</normaloff>:/new/prefix1/icon/delete.ico</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/icon/info.ico</normaloff>:/new/prefix1/icon/info.ico</iconset>
   </property>
   <property name="text">
    <string>
About</string>
   </property>
  </action>
  <action name="action_About_Qt">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/icon/Apps-Qt-icon.png</normaloff>:/new/prefix1/icon/Apps-Qt-icon.png</iconset>
   </property>
   <property name="text">
    <string>
About Qt</string>
   </property>
  </action>
  <action name="actionSet_Color_Window">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/icon/apariencia.ico</normaloff>:/new/prefix1/icon/apariencia.ico</iconset>
   </property>
   <property name="text">
    <string>Set Color Window</string>
   </property>
  </action>
  <action name="actionSet_Color_Text">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/prefix1/icon/color.ico</normaloff>:/new/prefix1/icon/color.ico</iconset>
   </property>
   <property name="text">
    <string>Set Color Text</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
